contrib/runit_boot.sh

```diff
@@ -38,8 +44,9 @@ echo "Started runsvdir, PID is $RUNSVDIR"
 echo "wait for processes to start...."
 
 sleep 5
-for _srv in $(ls -1 /etc/service); do
-    sv status $_srv
+mapfile -t etc_services < <(ls -1 /etc/service)
```

Comment:
this mapfile executes once when the bash script begins (at container launch). The array is initialized at that time.  This subtly changes the behaviour of "shutdown"

Reply:
I was not thinking about people changing services inside a running container so I didn't notice this subtlety, nice catch, thanks!